File: src/main/java/com/trip_service/trip_service/users/repositories/UserRepository.java
package com.trip_service.trip_service.users.repositories;

import com.trip_service.trip_service.users.models.Users;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.UUID;

@Repository
public interface UserRepository extends JpaRepository<Users, String> {

}

File: src/main/kotlin/com/trip_service/trip_service/core/users/repositories/UserRepository.kt
package com.trip_service.trip_service.core.users.repositories

import com.trip_service.trip_service.core.users.models.Users
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface UserRepository : JpaRepository<Users, String>  {
}